app/models/physical_ticket.rb

```diff
@@ -4,4 +4,12 @@ class PhysicalTicket < ActiveRecord::Base
   has_one :conference, through: :ticket_purchase
   has_one :user, through: :ticket_purchase
   has_many :ticket_scannings
+
+  after_create :send_ticket_confirmation_mail
+
+  def send_ticket_confirmation_mail
+    if conference.email_settings.send_on_ticket_confirmation?
+      Mailbot.ticket_confirmation_mail(conference, user, self).deliver_later
```

Comment:
We were talking about not sending two emails, as when paying the tickets there is an email already sent, what is the state of that?